ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionFragment.kt

```diff
@@ -207,7 +205,7 @@ class DataCollectionFragment : AbstractFragment(), BackPressListener {
       false
     } else {
       // Otherwise, select the previous step.
-      lifecycleScope.launch { viewModel.step(-1) }
```

Comment:
Perhaps for a follow up - why do we expose "step" with an arbitrary number of pages, rather than "next" and "previous"?

Reply:
Added a todo